daemons/gptp/common/ptp_message.cpp

```diff
@@ -1502,12 +1510,23 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	    Timestamp prev_peer_ts_theirs;
 	    FrequencyRatio rate_offset;
 	    if( port->getPeerOffset( prev_peer_ts_mine, prev_peer_ts_theirs )) {
+		    FrequencyRatio upper_limit, lower_limit;
+		    upper_limit = UPPER_FREQ_LIMIT*2;
+		    upper_limit /= US_PER_SEC;
+		    upper_limit += 1;
+		    lower_limit = UPPER_FREQ_LIMIT*2;
```

Comment:
I think keeping the global/#define in terms of PPM units makes them more intuitive. The math in this particular code maybe less so.

We could add a macro (inline function works too):

#define UPPER_LIMIT_PPM   250
#define LOWER_LIMIT_PPM -250
#define PPM_OFFSET_TO_RATIO(ppm) ((ppm)/((FrequencyRatio)US_PER_SEC)+1)

upper_ratio_limit = PPM_OFFSET_TO_RATIO(UPPER_LIMIT_PPM)
lower_ratio_limit = PPM_OFFSET_TO_RATIO(LOWER_LIMIT_PPM)